portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -1,27 +1,27 @@
 package com.google.sps.servlets;
```

Comment:
This file just has commented out code. Did you mean to remove this altogether?

Reply:
FormHandlerServlet contains work from week 2 so I didn't want to remove it altogether and lose that work.

Reply:
I'd suggest deleting this file. You can also go back to earlier changes from your master's commit history - https://github.com/cyber-mary/sps-portfolio/commits/main